terraform/per_account/deployable/lambda_cloudwatch_alarm_forwarder.tf

```diff
@@ -1,4 +1,5 @@
 resource "aws_lambda_function" "cloudwatch_alarm_forwarder_euw1_lambda" {
+  #checkov:skip=CKV_AWS_50:X-ray tracing is enabled for Lambda
```

Comment:
As this is repeated multiple, would it be better to use a global skip instead? 
https://github.com/bridgecrewio/checkov/blob/master/docs/2.Concepts/Suppressions.md#global-skip

Reply:
After running checkov and cheking the output I think this might be the better way of doing the exceptions.